CalendarApp/Logic Handlers/EventSyncHandler.m

```diff
@@ -136,20 +165,72 @@ - (void)syncDeleteToParse:(Event *)event
     }];
 }
 
-- (void)syncUpdateToParse:(Event *)event
-             remoteChange:(RemoteChange *)newChange{
+- (void)syncUpdateToParseWithEvent:(Event *)event {
+    [self.userData setObject:[NSDate date] forKey:@"lastUpdated"];
     [self.parseEventHandler updateEvent:event completion:^(BOOL success, NSString * _Nullable error) {
         if (!success) {
             // TODO: Error handling
+        }
+    }];
+}
+
+- (void)syncRemoteChangeToParseWithRemoteChange:(RemoteChange *)newChange {
+    [self.userData setObject:[NSDate date] forKey:@"lastUpdated"];
+    [self.parseChangeHandler addNewParseChange:newChange
+                                    completion:^(BOOL success, NSString * _Nullable error) {
+        if (!success) {
+            // TODO: save as local change?
```

Comment:
Good call. Would make sense to have these changes available for retries but it does add complexity.